provision-commerce-coordinator.sh

```diff
@@ -1,30 +1,9 @@
 name="commerce-coordinator"
 port="8140"
 
-docker-compose up -d --build
-
-# Install requirements
-# Can be skipped right now because we're using the --build flag on docker-compose. This will need to be changed once we move to devstack.
-
-# Wait for MySQL
-echo "Waiting for MySQL"
-until docker exec -i commerce-coordinator.db mysql -u root -se "SELECT EXISTS(SELECT 1 FROM mysql.user WHERE user = 'root')" &> /dev/null
-do
-  printf "."
-  sleep 1
-done
-sleep 5
-
-# Create the database
-docker exec -i commerce-coordinator.db mysql -u root -se "CREATE DATABASE commerce-coordinator;"
-
-# Run migrations
-echo -e "${GREEN}Running migrations for ${name}...${NC}"
-docker exec -t commerce-coordinator.app bash -c "cd /edx/app/${name}/ && make migrate"
-
 # Create superuser
 echo -e "${GREEN}Creating super-user for ${name}...${NC}"
-docker exec -t commerce-coordinator.app bash -c "echo 'from django.contrib.auth import get_user_model; User = get_user_model(); User.objects.create_superuser(\"edx\", \"edx@example.com\", \"edx\") if not User.objects.filter(username=\"edx\").exists() else None' | python /edx/app/${name}/manage.py shell"
```

Comment:
We may not want to delete these things, or, at least, we may want them back when we do move commerce-coordinator into a container

Reply:
I think leaving them commented-out in the file with a brief comment saying to put them back when we move development into docker would be best, it's easy to lose track of an external doc somewhere.

Reply:
ah I like that idea- thanks Phil!